app/views/partials/dashboard/_text_fields.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="">
+<p> <%= value %>
+
```

Comment:
Do not create empty components like that; if there's no css and simply a div and an element, it's not worth it. Is this properly used?